src/main/java/org/openstreetmap/atlas/checks/validation/tag/TollValidationCheck.java

```diff
@@ -132,19 +134,46 @@ public boolean validCheckForObject(final AtlasObject object)
                 .edgeProvingForwardIsEscapable(edgeInQuestion, alreadyCheckedObjectIds)
                 .orElse(null);
 
-        // Case three: tag modeling needs to be investigate on and around edge in question/proved
+        // Case three: tag modeling needs to be investigated on and around edge in question/proved
         // escapable routes
-        if (this.escapableEdgesNullChecker(escapableInEdge, escapableOutEdge) && this
-                .isCaseThree(edgeInQuestion, edgeInQuestionTags, escapableInEdge, escapableOutEdge))
+        if (escapableInEdge != null && escapableOutEdge != null && this.isCaseThree(edgeInQuestion,
+                edgeInQuestionTags, escapableInEdge, escapableOutEdge))
         {
-            markAsFlagged(object.getOsmIdentifier());
+            markAsFlagged(edgeInQuestion.getOsmIdentifier());
+            if (!isFlagged(escapableInEdge.getOsmIdentifier())
+                    && this.hasInconsistentTollTag(escapableInEdge))
+            {
+                markAsFlagged(escapableInEdge.getOsmIdentifier());
+                return Optional.of(this
+                        .createFlag(new OsmWayWalker(escapableInEdge).collectEdges(),
+                                this.getLocalizedInstruction(2, edgeInQuestion.getOsmIdentifier()))
+                        .addFixSuggestion(FeatureChange.add(
```

Comment:
I thought there wasn't going to be a fix suggestion for this case. Has that changed?

Reply:
This has been re-evaluated. At the beginning of this PR at the top I have the explanation in bold

Reply:
this one checks the edges that prove the edge in question is "escapable" but they are being passed through the "inconsistent" logic. That means that if they are found to be "inconsistent" we should flag them as that which means adding a fix suggestion.

Reply:
Ok, in that case can we add that to the documentation. Right now it only says cases 1 and 2 will have fix suggestions.

Reply:
that is still the case.. The edges proving escapable are put through case 2 before the edge in question can be deemed "escapable" in case 3. This is the discussion we had where the proving edges had to be put through the "inconsistent" test (case 2) first.

Reply:
Oh, ok. Sorry, I see how this works now. It is just a bit confusing because it is inside the case 3 if block.